deno/send_message/README.md

```diff
@@ -0,0 +1,102 @@
+# SendMessage() 
+
+A sample Deno Cloud Function sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
+
+
+_Example function payloads:_
+
+```json
+// Sending SMS
+{
+    "type":"SMS","receiver":"+123456789","message":"Programming is fun!"
+}
+// Sending Email
+{
+    "type":"EMAIL","receiver":"hello@example.com",
+    "message":"Programming is fun!","subject":"Programming is funny!"
+}
+// Sending message through Discord webhook 
+{
+    "type":"DISCORD","receiver":"","message":"Hi"
+}
+// Sending a tweet
+{
+    "type":"TWITTER","receiver":"","message":"Programming is fun!"
+}
+```
+
+
+_Successful function response:_
+
+
+```json
+{ "success" : true }
+```
+
+_Error function response:_
+
+
+```json
+{
+	"success": false,
+	"error": "Failed to send message, check the webhook URL"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+Mailgun
+* **MAILGUN_API_KEY** - API Key for Mailgun 
+* **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+Discord
+* **DISCORD_WEBHOOK_URL** - Webhook URL for Discord 
+
+Twilio 
+
+* **TWILIO_ACCOUNT_SID** - Acount SID from Twilio
+
+* **TWILIO_AUTH_TOKEN** - Auth Token from Twilio
+
+* **TWILIO_SENDER** - Sender Phone Number from Twilio
+
+Twitter
+
+* **TWITTER_API_KEY** - API Key for Twitter
+
+
+* **TWITTER_API_KEY_SECRET** - API Key Secret for Twitter
+
+* **TWITTER_ACCESS_TOKEN** - Access Token from Twitter
+
+* **TWITTER_API_KEY_SECRET** - Access Token Secret from Twitter
```

Comment:
```suggestion
* **TWITTER_ACCESS_TOKEN_SECRET** - Access Token Secret from Twitter
```

A duplicate of `TWITTER_API_KEY_SECRET`, should be `TWITTER_ACCESS_TOKEN_SECRET`.

Reply:
fixed!